manifests/tributequarg.yaml

```
@@ -0,0 +1,14 @@
+name: Tribute Quarg
+authors: Nova.
+homepage: https://github.com/Nova1422/plugins
```

You probably want this to point to
```suggestion
homepage: https://github.com/Nova1422/plugins/tree/main/myplugins/tributequarg
```